### DISCUSSION TITLE
Solved: Debian Linux: Audacious text very small

### DISCUSSION THREAD

**Discussion by Wm-A-Weezl** [audacious-media-player/audacious#1467] [Q&A] [answered]
I have 2 computers running the same version (12, "Bookworm") of Debian. Both have display size 1920x1080. Audacious has been installed in the laptop for several months and has always looked fine. I installed Audicious in a "thin client" (HP T620) today. The text of the title bar is normal, but all the text that Audacious itself displays is tiny - think 6-point, maybe smaller. It's usable, but with difficulty. Changing between GTK and Qt doesn't affect the text size. The Winamp view is _really_ small and still too small after control-D. How can I get Audacity to display text comparable to other programs? Thanks for ideas.

**Comment by radioactiveman** [comment]
> How can I get Audacity to display text comparable to other programs?

I hope you mean Audacious. :smile: Audacity is another application.

Screenshots would probably be useful here. And what resolution does the thin client have?

**Reply by Wm-A-Weezl** [reply]
Thanks for the reply, radioactiveman. Again, the resolution is 1920x1080. I'm placing a screenshot of Audacious next to Thunar file browser; Thunar shows the typical text size. But I just found something new. I opened KeePassX password manager, and its text is the same size as Audacious's. Other programs that I've used, e.g., Firefox and LibreOffice at the moment, show nice big text like Thunar. Clearly, the issue isn't strictly with Audacious, it's something that Audacious and KeePassX share. It now looks like a more general forum is the best place for this question. But any tips here are welcome. 
![Screenshot_2024-09-24_17-05-00](https://github.com/user-attachments/assets/42d192f3-2ec8-4e7b-86fb-4455bf809334)

**Comment by Wm-A-Weezl** [comment, marked as answer]
"LinuxOS" on the Debian forum provided the solution. [https://forums.debian.net/viewtopic.php?p=810965#p810965](url)
